src/platform/forms/tests/fixtures/mocks/mockUserSaved.js

```diff
@@ -26,13 +26,13 @@ const mockUser = {
       },
       in_progress_forms: [
         {
-          form: VA_FORM_IDS.FORM_10_10EZ,
-          last_updated: 1501608808,
+          form: VA_FORM_IDS.FORM_XX_123,
           metadata: {
-            last_updated: 1506792808,
-            expires_at: moment()
+            last_updated: 1506792, // unix time
+            expires_at: moment() // unix time
```

Comment:
I guess this means you know about my related PR? https://github.com/department-of-veterans-affairs/vets-website/pull/19873

Reply:
Haha yes I tried to mirror that with this code. My assumption is that your PR will be merged in prior to this one and I want to make sure that the updates you made are used in this PR

Reply:
I just merged that PR